tests/integration/test_ui.py

```diff
@@ -0,0 +1,57 @@
+import pathlib
+import textwrap
+import threading
+import time
+
+import dash  # noqa
+
+import ujt
+import ujt.main
+import ujt.server.server
+
+
+def test_main_ui(dash_duo):
+    # Start the reporting server
+    server_thread = threading.Thread(
+        target=ujt.server.server.serve,
+        args=(
+            "50052",
+            pathlib.Path(__file__).parent.absolute() / "data",
+        ),
+        daemon=True,  # kill the thread when the test stops
+    )
+    server_thread.start()
+
+    # initialize the ujt
+    ujt.config.REPORTING_SERVER_ADDRESS = "localhost:50052"
```

Comment:
maybe comment on why this is different from the default 50051 above

Reply:
will fix this